README.md

```diff
@@ -154,15 +166,15 @@ The complete YAML file is available at `~/tekton-pipeline/resources/git.yaml`. A
   kubectl apply -f resources/git.yaml
 ```
 
-**Create Tasks**
+### 3. Create tasks
 
-Task defines the steps of the pipeline. To deploy an application to cluster using source code in git repository, we define two tasks - `build-image-from-source` and `deploy-to-cluster`. In task defintion the parameters used as args are referred as `$(inputs.params.<var_name>)`.
+A *task* defines the steps of the pipeline. To deploy an application to a cluster using source code in the Git repository, we define two tasks &mdash; `build-image-from-source` and `deploy-to-cluster`. In task definition, the parameters used as args are referred as `$(inputs.params.<var_name>)`. <!--EM: What are "args"-->
```

Comment:
args -> arguments. It is used as args in configuration (yaml) file.

We can rephrase like - "If a task uses the parameters which needs to be replaced with the actual value during execution then those parameters are defined as `$(inputs.params.<var_name>)` in task definition.

Reply:
Okay, that sounds good.